autonomy.py

```diff
@@ -311,17 +312,15 @@ def update_thread(vehicle, address, autonomyToCV):
             autonomyToCV.xbeeMutex.acquire()
             send_till_ack(address, update_message, msg_id)
             autonomyToCV.xbeeMutex.release()
-
         time.sleep(1)
-
     change_status("ready")
 
 
 # Continuously sends message to given address until acknowledgement message is recieved with the corresponding ackid.
 def send_till_ack(address, msg, msg_id):
     # Instantiate a remote XBee device object to send data.
-    send_xbee = RemoteXBeeDevice(xbee, address)
-    packed_data = packb(json.dumps(ack), use_bin_type = True)
+    send_xbee = RemoteXBeeDevice(xbee, XBee64BitAddress.from_hex_string(address))
```

Comment:
XBee64BitAddress.from_hex_string(address)) to a local variable, this is pretty redundant